app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentPresenterImpl.kt

```diff
@@ -637,6 +637,13 @@ class EnrollmentPresenterImpl(
         }
     }
 
+    fun hasAccess(): Boolean {
+        if (getProgram().access()?.data()?.write() == true) {
+            return true
+        }
+        return false
```

Comment:
Can we do `return getProgram().access()?.data()?.write() ?: false` in here?

Reply:
yes I think it makes sense